sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/samples/java/com/azure/messaging/eventhubs/checkpointstore/blob/EventProcessorBlobCheckpointStoreSample.java

```diff
@@ -52,17 +52,17 @@ public static void main(String[] args) throws Exception {
             .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
             .buildAsyncClient();
 
-        EventProcessorBuilder eventProcessorBuilder = new EventProcessorBuilder()
+        EventProcessorClientBuilder eventProcessorClientBuilder = new EventProcessorClientBuilder()
             .connectionString(EH_CONNECTION_STRING)
             .consumerGroup("<< CONSUMER GROUP NAME >>")
             .processEvent(PARTITION_PROCESSOR)
             .processError(ERROR_HANDLER)
-            .eventProcessorStore(new BlobEventProcessorStore(blobContainerAsyncClient));
+            .checkpointStore(new BlobCheckpointStore(blobContainerAsyncClient));
 
-        EventProcessor eventProcessor = eventProcessorBuilder.buildEventProcessor();
-        eventProcessor.start();
+        EventProcessorClient eventProcessorClient = eventProcessorClientBuilder.buildEventProcessorClient();
+        eventProcessorClient.start();
         TimeUnit.MINUTES.sleep(5);
```

Comment:
nit: Add a comment that we wanted to simulate the customer's program doing other work until it is ready to stop.